pvlib/iam.py

```diff
@@ -92,7 +145,7 @@ def ashrae(aoi, b=0.05):
     return iam
 
 
-def physical(aoi, n=1.526, K=4.0, L=0.002, *, n_ar=None):
```

Comment:
All this does is require that the `n_ar` argument be named, which seems like an antipattern for this codebase.